src/main/kotlin/de/cyface/uploader/DefaultUploader.kt

```diff
@@ -78,6 +78,7 @@ class DefaultUploader(private val apiEndpoint: String) : Uploader {
         jwtToken: String,
         metaData: RequestMetaData,
         file: File,
+        endpoint: URL,
```

Comment:
Why do you specify endpoint here, in addition to the apiEndpoint property of the class?